tests/ValueObject/ExportConfigTest.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Tests\ValueObject;
+
+use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\IncrementalFetchingConfig;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\InputTable;
+use Keboola\DbExtractorConfig\Exception\InvalidArgumentException;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\ValueObject;
+use PHPUnit\Framework\Assert;
+
+class ExportConfigTest extends BaseValueObjectTest
+{
+    public function createValueObjectFromArray(array $properties): ValueObject
+    {
+        $query = $properties['query'];
+        $table = $properties['table'];
+        $incrementalFetchingConfig = $properties['incrementalFetchingConfig'];
+        $columns = $properties['columns'];
+        $outputTable = $properties['outputTable'];
+        $primaryKey = $properties['primaryKey'];
+        $maxRetries = $properties['maxRetries'];
+        return new ExportConfig(
+            $query,
+            $table,
+            $incrementalFetchingConfig,
+            $columns,
+            $outputTable,
+            $primaryKey,
+            $maxRetries,
+        );
+    }
+
+    public function getAllProperties(): array
+    {
+        return [
```

Comment:
tím, že je to tady vyjmenované tak ten test IMHO nic neodhalí - přidáš novou property a pokud ji nepřidáš do testu, tak to nefailne. Ne?

Reply:
rovnako ako ked pridas property a nenapises na nu test ... nevidim v tom problem, to je otazka review, ze nieco nie je pokryte testom ... rozmyslal som aj nad moznostov nacitat vsetky property cez reflection a porovnat to, ... ale podla mna by ta to velmi obmedzovalo, ak by si nejaku property potreboval vyriesit nejak inak ako ostatne.

Reply:
Chápu. Já se snažím psát testy tak, aby byly křehké a když někde něco upravím, tak aby se rozbily a donutili mě zkontrolovat, co přesně jsem rozbil.

Reply:
jasne, no v tomto konkretnom pripade si myslim, ze (unit!) test, ktory by si napisal, by tiez nespadol preto, ze si tam pridal novu property, ktora s povodnymi testami a kodom nie je prepojena ... a inak v tomto pripade by to spadlo na konstruktore, ak by to bol teda povinny parameter ....